### ISSUE TITLE
🧐[问题] CustomContent for label

### ISSUE THREAD

**Issue by philxo** [ant-design/ant-design-charts#1267]
I use the content property in label so I can put in something specifically for the label. But this only accepts a string. I want to add a bit of styling to the labels of the bars. I use custom content for the tooltip. Is there anything similar for the labels? I see the formatter property, but I'm not too sure what it does.

**Comment by philxo** [comment]
For more information, the things I specifically want to do is add text-overflow: ellipsis; and make part of the text bold.

**Comment by lxfu1** [comment]
Ellipsis is can't supported. @visiky 
```ts
 label: {
  //  'top', 'bottom', 'middle',
  position: 'middle',
  style: {
    fill: '#f20',
    fontWeight:'bold',
    fontSize: 20,
  },
},
```

**Comment by visiky** [comment]
- Label ellipsis is not supported for every plot (except `Pie`) yet, there are 2 ways to support it:
1. Maybe you can use `label.formatter` to customize the maxLimitLength of label.
2. Using the advanced way to `registerLabelLayout`, to customize the layout of labels. 👉 [API](https://g2.antv.vision/en/docs/api/advanced/register-label#%E8%87%AA%E5%AE%9A%E4%B9%89-label-%E5%B8%83%E5%B1%80%E5%87%BD%E6%95%B0)

- we'll support it after the dependency of @antv/g2 upgrading to the new version, which will be released 5.0 this year.
